.github/workflows/sdk_build.yml

```diff
@@ -60,11 +60,28 @@ jobs:
           path: firebase-cpp-sdk
           submodules: true
 
+      - name: Support longpaths
+        if: startsWith(matrix.os, 'windows')
+        run: git config --system core.longpaths true
+
+      # Set up the requirements, and install Unity
```

Comment:
Can you add a comment explaining that this needs to be done before the python setup, just to keep it clear.